tox.ini

```diff
@@ -45,7 +45,7 @@ exclude = venv,.git,.tox,dist,*egg,cachito/web/migrations,.env
 ignore = D100,D104,D105,W503,E203
 per-file-ignores =
     # Ignore missing docstrings in the tests and migrations
-    tests/*:D103
+    tests/*:D101,D102,D103
```

Comment:
This makes flake8 also ignore docstrings for classes and public methods in the test folder.